src/stars.coffee

```diff
@@ -60,8 +61,24 @@ class Stars extends Command
   installPackages: (packages, callback) ->
     return callback() if packages.length is 0
 
-    commandArgs = packages.map ({name}) -> name
-    new Install().run({commandArgs, callback})
+    toName = ({name}) -> name
+
+
+    list = new List()
+    listOptions =
+      argv:
+        dev: true
+        bare: true
+
+    list.listDevPackages listOptions, (error, devPackages) ->
```

Comment:
I misunderstood how dev packages worked. I thought it was a problem to have the same package in both. I'll update this to not check dev packages.